libs/wingsdk/src/shared-aws/function.inflight.ts

```diff
@@ -22,7 +22,7 @@ export class FunctionClient implements IFunctionClient {
   public async invoke(payload?: string): Promise<string | undefined> {
     const command = new InvokeCommand({
       FunctionName: this.functionArn,
-      Payload: payload ? fromUtf8(JSON.stringify(payload)) : undefined,
+      Payload: fromUtf8(payload ? JSON.stringify(payload) : "null"),
```

Comment:
aws needs a json value, otherwise the handler argument is `{}`

using the json `null` makes sense to speak AWS's language here